src/screens/help.test.ts

```diff
@@ -185,6 +185,21 @@ describe('#helpScreen()', () => {
 		expect(result).toContain('Just used for testing (accepts: a2, b2, c2, d2)');
 	});
 
+	it('Emits help with dynamic async accepts undefined properties', async () => {
+		process.argv = [
+			'/path/to/node',
+			path.join(testProjectsPath, 'pizza-ordering', 'cli', 'entry.js'),
+			'order',
+			'dynamic-async-accepts-undefined',
+			'--help',
+		];
+
+		const result = removeFormatting(await helpScreen());
+
+		expect(result).toContain('What type of pizza to order');
+		expect(result).toContain('Just used for testing');
```

Comment:
I would expect there to be a `.not.toContain()` in this type of test, since we want to make sure there _isn't_ an "accepts:" note when we return `undefined`.